VerloopSDK/VerloopSDK.swift

```diff
@@ -145,7 +146,7 @@ import Foundation
             verloopNavigationController?.view.backgroundColor = .white
 
             verloopNavigationController?.navigationBar.barTintColor = UIColor.white
-
+            verloopController?.setViewControllerLifeCycleDelegate(manager)
```

Comment:
You could've directly set the manager as delegate in the VLViewController initialisation, and not as a seperate method.

Reply:
done